sources/Algo/include/Algo.h

```diff
@@ -323,20 +323,49 @@ struct HvdcLineDefinition {
   const ConverterId converter2_id;                             ///< second converter id
   const BusId converter2_busId;                                ///< second converter bus id
   const boost::optional<bool> converter2_voltageRegulationOn;  ///< second converter voltage regulation parameter, for VSC converters only
-  const Position position;                                     ///< position of the converters of this hvdc line compared to the main connex component
+  mutable Position position;                                   ///< position of the converters of this hvdc line compared to the main connex component
```

Comment:
mutable is only for const objects that must be modified. This is not the case here.

Reply:
It was useful for the set of the find function, this function returned a const iterator and I needed to modify the position of the element pointed by the iterator. Hence the mutable.(thanks stackoverflow).
But with the map it won't be needed anymore as it does not return a const iterator